storyblok-vercel-configuration.md

```diff
@@ -0,0 +1,111 @@
+# Production, Staging, Preview configuration
+
+Storyblok = place where content is stored (Headless CMS = content management
+system)
+
+GitHub (this repo) = place where code is stored (hosted SVC = software version
+control)
+
+Vercel/Next.js = service that generates and deploys website by combining content
+and code.
+
+The following document uses Quansight Consulting "LLC" as an example, but it
+also applies to Labs: wherever you come across the string `consulting` below,
+you just have to replace it with `labs`.
+
```

Comment:
Should probably add a paragraph here explaining that even though the sites have separate instances in Storyblok and Vercel, they share the same repo. So merging to main branch of this repo triggers production deploy for both Labs and Consulting. 

This is mildly annoying because it means that if we:

1. Have changes on LLC staging site that we don't want in production (quansight.com) yet, and
2. Have code changes in `develop` branch for the Labs site that we want to get into production, then
3. We will be blocked from updating labs.quansight.org

I think this means we must have a rule that says: never publish anything from Storyblok, and never merge anything into `develop` branch unless it's 100% ready to go to the main production site.

Reply:
Which is a pretty standard rule for staging sites anyway, I believe

Reply:
Could this be (partially) solved by instituting `develop-labs` and `develop-consulting` branches? I'm guessing not, since then we would no longer have a single source of truth to use for `staging`?

Reply:
Please no more branches - I believe the setup is already complex enough (assessed based on the many many many questions and unknowns we have unveiled).